src/Identity/IdentityServer/BaseRequestValidator.cs

```diff
@@ -96,11 +96,12 @@ public abstract class BaseRequestValidator<T> where T : class
         _distributedCache = distributedCache;
         _cacheEntryOptions = new DistributedCacheEntryOptions
         {
-            // This sets the time an item is cached to 15 minutes. This value is hard coded 
+            // This sets the time an item is cached to 17 minutes. This value is hard coded
             // to 15 because to it covers all time-out windows for both Authenticators and
```

Comment:
nit: 
```suggestion
            // This sets the time an item is cached to 17 minutes. This value is hard coded
            // to 17 because to it covers all time-out windows for both Authenticators and
```

Reply:
Nooooo xD

Reply:
@jlf0dev Fixed!